src/condor_utils/file_transfer.cpp

```diff
@@ -6033,6 +6077,37 @@ int FileTransfer::OutputFileTransferStats( ClassAd &stats ) {
 	// back to previous priv state
 	set_priv(saved_priv);
 
+	// In addition to the log file, we also want to save stats in our FileTransferInfo object
+	std::string protocol;
+	if (stats.LookupString("TransferProtocol", protocol)) {
+		// Do not record cedar stats here, only plugins
+		if (protocol != "cedar") {
+			upper_case(protocol);
+			std::string attr_count = protocol + "FilesCount";
+			std::string attr_size = protocol + "SizeBytes";
+
+			int num_files;
+			if (!Info.stats.LookupInteger(attr_count, num_files)) {
+				num_files = 0;
+			}
+			num_files++;
+			Info.stats.InsertAttr(attr_count, num_files);
+
+			int this_size_bytes;
+			if (stats.LookupInteger("TransferTotalBytes", this_size_bytes)) {
+				int prev_size_bytes;
+				if (!Info.stats.LookupInteger(attr_size, prev_size_bytes)) {
+					prev_size_bytes = 0;
+				}
+				Info.stats.InsertAttr(attr_size, prev_size_bytes + this_size_bytes);
+			}
+		}
+	}
+
+	classad::ClassAdUnParser unparser;
+	std::string stats_str;
+	unparser.Unparse(stats_str, &Info.stats);
```

Comment:
What is the point of this code?

Reply:
This was some debug code I forgot to remove, it's gone now.